matRadGUI.m

```diff
@@ -115,6 +115,16 @@ function matRadGUI_OpeningFcn(hObject, ~, handles, varargin)
 axis equal off;
 set(f, 'AlphaData', alpha);
 
+% turn off the datacursormode (since it does not allow to set scrolling
+% callbacks
+handles.dcm_obj = datacursormode(handles.figure1);
+set(handles.dcm_obj,'DisplayStyle','window');
```

Comment:
This line is debatable... instead of showing the values directly besides the cursor, it opens a small window containing information that can be moved, which is useful due to the amount of information that can be present in a datatip. Restricts usage to only 1 datapoint however... so this line could also be removed. (This option can be changed by right-clicking on the datatip, btw).